webapp/src/errorList.tsx

```diff
@@ -42,16 +48,17 @@ export class ErrorList extends React.Component<ErrorListProps, ErrorListState> {
 
     render() {
         const {isCollapsed, errors, exception} = this.state;
-        const errorsAvailable = !!errors?.length || !!exception;
-        const collapseTooltip = lf("Collapse Error List");
 
+        let errorCount = 0;
+        (errors || []).forEach(e => errorCount += e.count);
```

Comment:
nit: ``(errors || []).reduce((e, count) => e.count + count)``

Reply:
> nit: `(errors || []).reduce((e, count) => e.count + count)`

Delayed grouping made this unnecessary. Reverted my changes in this method.